src/components/visualizations/chart/highcharts/test/getOptionalStackingConfiguration.spec.ts

```diff
@@ -303,6 +303,123 @@ describe("getOptionalStackingConfiguration", () => {
             expect(result).not.toHaveProperty("yAxis.stackLabels");
         });
 
+        describe("Stack Measures in Combo chart", () => {
+            const chartOptions = {
+                type: VisualizationTypes.COMBO,
+                yAxes: [{}, {}],
+            };
+
+            it.each([
+                ["stackMeasures", VisualizationTypes.COLUMN, NORMAL_STACK],
+                ["stackMeasures", VisualizationTypes.AREA, NORMAL_STACK],
+                ["stackMeasuresToPercent", VisualizationTypes.COLUMN, PERCENT_STACK],
+                ["stackMeasuresToPercent", VisualizationTypes.AREA, PERCENT_STACK],
+            ])("should return series with %s config if seriesType=%s", (stackConfig, type, stackType) => {
```

Comment:
it should be `should return series with %s config if series type is %s`
and we should add type to params